### ISSUE TITLE
RemovedInDjango19Warning: django.utils.importlib will be removed in Django 1.9.

### ISSUE THREAD

**Issue by facconi** [aldryn/aldryn-search#51]
I receive the following warning every time I start the server:

```
site-packages/aldryn_search/utils.py:16: RemovedInDjango19Warning: 
             django.utils.importlib will be removed in Django 1.9.
from django.utils.importlib import import_module
```

Using:

* Django==1.8.6
* aldryn-search==0.2.10

**Comment by czpython** [comment]
Thanks. Fixed by 4f8f0d87a218a67bfe9c1304e4c1c5651e52c1b2

**Comment by facconi** [comment]
Thanks!
